File: Programmers/NOMORE12/12921.js
function solution(n) {
  var answer = 0;
  for (let i = 2; i <= n; i++) {
    answer = isPrime(i) ? answer + 1 : answer;
  }

  function isPrime(number) {
    for (let i = 2; i * i <= number; i++) {
      if (number % i === 0) return false;
    }
    return true;
  }

  return answer;
}

console.log(solution(2500));

File: Programmers/HEROHJK/12921.swift
import Foundation

func solution(_ n:Int) -> Int {
    var count = 0
    
    (2 ... n).forEach { number in
        if isPrimeNumber(at: number) { count += 1 }
    }
    
    return count
}

func isPrimeNumber(at: Int) -> Bool {
    let root = Int(sqrt(Double(at)))
    if 2 > root { return true }
    
    for number in 2 ... root {
        if at % number == 0 { return false }
    }
    
    return true
}